packages/astro/config.d.ts

```diff
@@ -401,6 +401,23 @@ export interface PlatformaticAstroConfig {
             [k: string]: unknown;
           };
     };
+    verticalScaler?: {
+      enabled?: boolean;
+      maxWorkers?: number;
+      scaleUpELU?: number;
+      scaleDownELU?: number;
+      minELUDiff?: number;
+      timeWindowSec?: number;
+      cooldownSec?: number;
+      scaleIntervalSec?: number;
+      applications?: {
```

Comment:
This seems odd. Why is applications being added here?

Reply:
This is an object where you can specify an application specific scaler options like: `minWorkers`, `maxWorkers`.

Reply:
Agreed. But this include the full list of applications again.